tensorflow_lite_support/python/task/processor/processor_options.py

```diff
@@ -11,23 +11,69 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Options to configure Embedding APIs."""
+"""Options to configure Vision APIs."""
 
 import dataclasses
-from typing import Optional
+from typing import Optional, List
 
 
 @dataclasses.dataclass
 class EmbeddingOptions:
   """Options for embedding processor."""
-  # Whether to normalize the returned feature vector with L2 norm. Use this
-  # option only if the model does not already contain a native L2_NORMALIZATION
-  # TF Lite Op. In most cases, this is already the case and L2 norm is thus
-  # achieved through TF Lite inference.
+
+  """
```

Comment:
Sorry, I misunderstood the requirements for Python docstring. Each attributes should be documented in the class definition as below.

https://github.com/tensorflow/tensorflow/blob/d248815ff16901941ca617b020a26739f0cce230/tensorflow/lite/python/lite.py#L188